lib/active_fedora/clean_connection.rb

```diff
@@ -0,0 +1,35 @@
+module ActiveFedora
+  class CleanConnection < SimpleDelegator
+    def get(*args)
+      result = __getobj__.get(*args) do |req|
+        prefer_headers = Ldp::PreferHeaders.new(req.headers["Prefer"])
+        prefer_headers.omit = prefer_headers.omit | omit_uris
+        req.headers["Prefer"] = prefer_headers.to_s
+      end
+      CleanResult.new(result)
+    end
+
+    private
+
+    def omit_uris
+      [
+        RDF::Fcrepo4.ServerManaged,
+        RDF::Ldp.PreferContainment,
+        RDF::Ldp.PreferEmptyContainer,
+        RDF::Ldp.PreferMembership
+      ]
+    end
+
+    class CleanResult < SimpleDelegator
+      def graph
+        __getobj__.graph.delete(has_model_query)
```

Comment:
This is a bit odd. Does the `graph#delete` return a graph object that no longer has the specified triple?

I would expect:

```ruby
def graph
  __getobj__.graph.delete(has_model_query)
  __getobj__.graph
end
```

This change does not rely on the overloaded `graph#delete` method that is acting as both a command and a query method.